### PR TITLE
Node v6

### LINKED ISSUE #11: Bump pg version to support Node v6
In order to support last LTS of Node.js (v6) we need to bump `pg` version to 6.x

cc @rochoa

### PR DESCRIPTION
**Pull request by rochoa**
This closes https://github.com/CartoDB/node-cartodb-psql/issues/11.

### CODE REVIEW

File: lib/psql.js [hunk 1]
```diff
@@ -256,7 +262,7 @@ var PSQL = function(connectionParams, poolParams, options) {
     var keepAliveConfig;
 
     if (globalSettings.db_keep_alive) {
-        keepAliveConfig = _.extend({}, globalSettings.db_keep_alive);
```
Comment (rochoa) [inline lib/psql.js:259]: Why did we go for just `enabled`?

If we don't have support for keep alive in the underlying library I would remove this.

File: lib/psql.js [hunk 2]
```diff
@@ -339,7 +345,7 @@ var PSQL = function(connectionParams, poolParams, options) {
             }
             var types = ["'geometry'","'raster'"]; // types of interest
             var typesSqlQuery = "SELECT oid, typname FROM pg_type where typname in (" + types.join(',') + ")";
-            client.query(typesSqlQuery, function(err,res) {
+            client.query(typesSqlQuery, [], function(err,res) {
```
Comment (rochoa) [inline lib/psql.js]: It seems like client.query still supports (sql, callback) signature, doesn't it?

File: lib/psql.js [hunk 3]
```diff
@@ -410,18 +416,24 @@ var PSQL = function(connectionParams, poolParams, options) {
...
+                    query.emit('error', new Error('Row too large, was ' + message.length + ' bytes'));
                 });
 
                 // NOTE: for some obscure reason passing "done" directly
                 //       as the listener works but can be slower
                 //      (by x2 factor!)
                 query.on('end', function() {
                     client.removeListener('notice', noticeListener);
-                    done(options.destroyOnError && errorHappened);
+                    done(false);
```
Comment (rochoa) [inline lib/psql.js]: Should we call `done` without any param?

**Comment by dgaubert** [comment]
👍

**Review by dgaubert — APPROVED** [review]
LGTM

**Comment by rochoa** [comment]
But, can you review my comments?

**Comment by dgaubert** [comment]
> But, can you review my comments?

comments?? where??

**Comment by rochoa** [comment]
I didn't "commit" my CR and my comments were "drafts". Sorry.

File: lib/psql.js (hunk 1 above)
Comment (dgaubert) [reply lib/psql.js:259]: pg@6.x has support for keep alive by config but t just need a boolean value instead a object with two properties. I wanted to keep this, but if it is not needed I could remove it. See: https://github.com/brianc/node-postgres/blob/master/lib/connection.js#L59-L61

File: lib/psql.js (hunk 2 above)
Comment (dgaubert) [reply lib/psql.js]: Humm, If it does, we should keep previous signature. On it.

File: lib/psql.js (hunk 3 above)
Comment (dgaubert) [reply lib/psql.js]: It's going to work, if it receives a `falsy` value then the client won't be released. See: https://github.com/brianc/node-pg-pool/blob/master/index.js#L97-L106

So, if you prefer call it without any param then fine by me.

File: lib/psql.js (hunk 1 above)
Comment (rochoa) [reply lib/psql.js:259]: Thank you for the clarification.

**Comment by rochoa** [comment]
🇪🇸  Vamos!

**Pull request merged**